### PR TITLE
Add interface tests

### PR DESCRIPTION
**Pull request by arturo-seijas**
Applicable spec: N/A

### Overview

Add the configuration needed for interface tests

### Rationale

N/A

### Juju Events Changes

N/A

### Module Changes

N/A

### Library Changes

N/A

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -101,6 +104,15 @@ deps =
 commands =
     bandit -c {toxinidir}/pyproject.toml -r {[vars]src_path} {[vars]tst_path}
 
+[testenv:interface]
```
Comment (Thanhphan1147) [inline tox.ini:107]: Do you know whether we run this target in GH workflow?

**Review by Thanhphan1147 — DISMISSED** [review]

**Review by yanksyoon — DISMISSED** [review]
TIL! Thank you for the generous comment!

File: tox.ini (hunk 1 above)
Comment (arturo-seijas) [reply tox.ini:107]: We run all the default targets, that is, the ones defined as `envlist` at the top of the `tox.ini` file

**Review by Thanhphan1147 — DISMISSED** [review]
LGTM

**Review by gtrkiller — DISMISSED** [review]
LGTM

**Review by gtrkiller — APPROVED** [review]
LGTM

**Review by Thanhphan1147 — APPROVED** [review]

**Pull request merged**